sklearn/inspection/_partial_dependence.py

```diff
@@ -590,9 +590,11 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
     from matplotlib import transforms  # noqa
     from matplotlib.ticker import MaxNLocator  # noqa
     from matplotlib.ticker import ScalarFormatter  # noqa
-
     # set target_idx for multi-class estimators
-    if hasattr(estimator, 'classes_') and np.size(estimator.classes_) > 2:
+    # TODO: Remove isinstance check in 0.24
```

Comment:
Maybe be slightly more explicit in the comment as the deprecation warning skipping logic is not trivial. For instance

```
# Attribute classes_ in DecisionTreeRegressor has been moved to super class
# BaseDecisionTree and is deprecated. The isinstance(estimator, DecisionTreeRegressor)
# check is used to prevent raising a spurious warning. 
# TODO: Remove isinstance check in 0.24 when the `classes_` attribute is actually
# removed from the DecisionTreeRegressor class.
```